static/templates/message_controls.hbs

```diff
@@ -1,18 +1,34 @@
 <div class="message_controls{{#status_message}} sender-status-controls{{/status_message}} no-select">
     {{#if msg/sent_by_me}}
-    <div class="edit_content message_control_button"></div>
+    <div class="edit_content message_control_button" data-tooltip-template-id="edit-content-template"></div>
+    <template id="edit-content-template">
+        {{#tr}}Edit{{/tr}}
+        {{hotkey_hints "E"}}
+    </template>
+    <template id="view-source-template">
+        {{#tr}}View source{{/tr}}
+        {{hotkey_hints "E"}}
+    </template>
```

Comment:
Somehow this ends up sometimes showing the "View source" text with the "Move message" icon...

Reply:
Fixed this issue with the most recent commits, the issue was a combination of `move message` option not being taken into account and the change in the name of the boolean variable from `is_editable` to `is_content_editable` due to a recent merge.